### ISSUE TITLE
Feature request: Indicate days or dates with scheduled events

### ISSUE THREAD

**Issue by yeboahnanaosei** [elementary/wingpanel-indicator-datetime#54]
Currently, there is no way to tell from the wingpanel which dates have scheduled events. To tell that a date has a scheduled event, you will have to open the calendar app itself to know.

I imagine that as a cool feature, dates that have scheduled events can be 'boldened' so that with a quick glance, the user is able to at least have an idea of days with scheduled events.

So assuming you have scheduled events for the 20th and 27th, those dates can be 'boldened'; then upon double clicking on that date, the calendar app launches for you to view the details of the event (which is the way it currently behaves anyway).
 
This is just how I am imagining it. Perhaps a better and cleaner design approach can work. 
**The bottom line is that, I think it will be nice if the wingpanel calendar could indicate which dates have scheduled events**
![indicate-scheduled-event](https://user-images.githubusercontent.com/22800157/34637293-ca14c1f6-f2aa-11e7-908d-6a5509736b98.png)

**Comment by yeboahnanaosei** [comment]
Closing because I just realized its a duplicate of #13
